vite.config.js

```diff
@@ -4,24 +4,28 @@ import laravel from 'laravel-vite-plugin'
 import { createVuePlugin } from 'vite-plugin-vue2'
 import { visualizer } from 'rollup-plugin-visualizer'
 
-export default defineConfig({
-    plugins: [
-        laravel({
-            input: [
-                'resources/css/app.css',
-                'resources/js/app.js',
-            ],
-            refresh: true,
-        }),
-        createVuePlugin(),
-        visualizer(),
-    ],
-    resolve: {
-        preserveSymlinks: true,
-        alias: {
-            '@': path.resolve(__dirname, './resources/js'),
-            'Vendor': path.resolve(__dirname, './vendor'),
-            'vue': path.resolve(__dirname, './node_modules/vue/dist/vue.js')
+export default ({ mode }) => {
+    let vueProduction = process.env?.NODE_ENV === 'production' ? '.min' : '';
```

Comment:
Maybe use the debug boolean from the env for this? As on test environments everything is also compiled as production but the env is or should be staging.